.github/workflows/main.yml

```diff
@@ -37,3 +37,15 @@ jobs:
         with:
           go-version: ${{ matrix.go_version }}
       - run: go test -v -race ./...
+  format:
+    name: Check formatting of code snippets in markdown files
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
+      - run: npm install -g mdsf-cli
+      - run: ./.ci.gofmt.sh
+      - run: ./.ci.readme.fmt.sh
```

Comment:
Does this directly call the gofmt tool? If so then this should be under the jobs.build key so that it always runs with the version/s of go whose format we target.

Reply:
Moved to `jobs.build`